tests/system/import_test.go

```diff
@@ -12,23 +15,71 @@ func TestImport(t *testing.T) {
 	t.Parallel()
 
 	s := NewSystemTest(t, "cases/import", nil)
+
+	// Don't actually run this test if dryrun is specified
+	// as it will first create resources then test imports
+	if s.Terraform.DryRun() {
```

Comment:
not sure why this was added when that is already accessible via the `dry   = flag.Bool(...)`. 
Also, test should be skipped using the `t.Skip` function, e.g.:
```
if *dry {
    t.Skip("Test cannot execute during a dry run")
}
```

Reply:
Because we don't pass in the cli flag or context into each system test, I can't check on the CLI flag content within the test. So I can't directly test for the contents of `dry` or `flag`. This test is unique specifically because it creates resources first and then uses the `*SystemTest`'s tests.

Reply:
The only way I've been able to see to infer that the dry flag has been specified is through `*SystemTest.Terraform.DryRun()`

Reply:
the dry flag is a global variable, so you can check on it form each system test.

Reply:
If I'm importing the flag package here, could I do something like:
```
     // Don't actually run this test if dryrun is specified
     // as it will first create resources then test imports
     if flag.Lookup("dry") {
         t.Skipf("Test cannot execute during a dry run")
     }
```